src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import {
+  Redirect,
+} from 'react-router-dom';
+
+import processBadResponse from '../../processBadResponse';
+import { defaultErrors } from '../../constants';
+import ForgotPasswordForm from './ForgotPasswordForm';
+import useForgotPasswordMutation from './useForgotPasswordMutation';
+
+const ForgotPassword = () => {
+  const [userEmail, setUserEmail] = useState(null);
```

Comment:
you don't need this state, instead of Redirect usage you can call `history.push/replace` in `onSubmit`